src/app/main.cpp

```diff
@@ -49,6 +91,8 @@ int main( int argc, char **argv )
 
 #ifdef ANDROID
   QgsApplication app( argc, argv, true, AndroidPlatformUtilities().packagePath() + QStringLiteral( "/resources" ) );
+  qInstallMessageHandler(qfMessageHandler);
```

Comment:
```suggestion
#ifdef ANDROID
  qInstallMessageHandler(qfMessageHandler);
#endif
```

Reply:
Why adding the ifdef here, I'm already in a ifdef ANDROID (check line 92)